recipe/meta.yaml

```diff
@@ -1,15 +1,12 @@
-{% set version = "0.29.27" %}
+{% set version = "0.29.28" %}
 
 package:
   name: cython
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/C/Cython/Cython-{{ version }}.tar.gz
-  sha256: c6a442504db906dfc13a480e96850cced994ecdc076bcf492c43515b78f70da2
-  patches:
-    # xref: https://github.com/cython/cython/pull/4236
-    - patches/pypy37-eval.patch
```

Comment:
Dropped as this was included in commit ( https://github.com/cython/cython/commit/2d287a6191bbba1e120ba9240b24ce364c443204 ) per this comment ( https://github.com/cython/cython/pull/4236#issuecomment-1029831500 )